drizzlepac/haputils/poller_utils.py

```diff
@@ -868,8 +868,9 @@ def build_poller_table(input, log_level, poller_type='svm'):
                             poller_row['skycell_obj'] = scell_obj
                             # append new row to table
                             new_poller_table.add_row(poller_row[0])
-
-    return new_poller_table
+        poller_table = new_poller_table
```

Comment:
Getting confused between all these PRs.  This line needs to be indented.  I also think Mike addressed this in his PR.

Reply:
Never mind.  This is OK.